statistics/handle/handle.go

```diff
@@ -952,7 +952,7 @@ func (h *Handle) extendedStatsFromStorage(reader *statsReader, table *statistics
 }
 
 // SaveStatsToStorage saves the stats to storage.
-func (h *Handle) SaveStatsToStorage(tableID int64, count int64, isIndex int, hg *statistics.Histogram, cms *statistics.CMSketch, topN *statistics.TopN, fms *statistics.FMSketch, statsVersion int, isAnalyzed int64) (err error) {
+func (h *Handle) SaveStatsToStorage(tableID int64, count int64, isIndex int, hg *statistics.Histogram, cms *statistics.CMSketch, topN *statistics.TopN, fms *statistics.FMSketch, statsVersion int, isAnalyzed int64, isPartition bool, dynamicPrune bool) (err error) {
```

Comment:
How about combining these two variables into one `needGlobalStats` = `dynamicPrune && isPartition`?

Reply:
Ignore this, using two variables is OK.

Reply:
Why? :rofl:  The only usage is the `isPartition && dynamicPrune`

Reply:
updated